python/cudf_polars/tests/conftest.py

```diff
@@ -1,15 +1,49 @@
-# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES.
 # SPDX-License-Identifier: Apache-2.0
 from __future__ import annotations
 
 import pytest
+from dask.distributed import Client, LocalCluster
```

Comment:
Can we make this import conditioned on actually getting the appropriate options.

Reply:
Good catch, done in https://github.com/rapidsai/cudf/pull/17364/commits/718b7b86dd153d0426365f3ff4ab91b8bc273ab1 .